tls/s2n_security_policies.c

```diff
@@ -615,6 +615,42 @@ const struct s2n_security_policy security_policy_pq_tls_1_3_2023_06_01 = {
     .ecc_preferences = &s2n_ecc_preferences_20201021,
 };
 
+/* Same as security_policy_pq_tls_1_2_2023_04_07, but with updated KEM prefs */
+const struct s2n_security_policy security_policy_pq_tls_1_3_2023_10_07 = {
```

Comment:
nit: Sorry, could you also update the struct names, just so it's consistent with the version strings?

Reply:
ah, whoops. yes will do.